tests/gp/test_hsgp_approx.py

```diff
@@ -194,17 +197,20 @@ def test_conditional(self, model, cov_func, X1, parameterization):
         prior and compare them using MMD two sample test.  Tests both centered and non-centered
         parameterizations.  The conditional should match the prior when no data is observed.
         """
+        seeds = np.arange(5) + 197  # 5 possible seeds
+        rng = np.random.default_rng(np.random.choice(seeds))
+
         with model:
             hsgp = pm.gp.HSGP(m=[100], c=2.0, parameterization=parameterization, cov_func=cov_func)
```

Comment:
try increasing `m` here to 500 too.  This got rid of the random failures for me locally